ECommerceDemoCommon/Contracts/IDataProvider.cs

```diff
@@ -1,17 +1,19 @@
 ﻿using System.Collections.Generic;
 
-namespace ECommerceDemoWebAPI.Contracts
+namespace ECommerceDemoCommon.Contracts
```

Comment:
Common is a hazardous word in software development. To me, Common means that The Great Lord is here.